Code/Python/kadane_algorithm.py

```diff
@@ -0,0 +1,57 @@
+#kadane's algorithm 
+#Given any array of integers, the algorithm to find the maximum subarray sum is called Kadane’s Algorithm.
+#Problem statement: Given Array Of N integers a1, a2, a3, ….. aN, find the maximum sum of sub-array of a given array.
+
+#The basic idea of kadane's algorithm is to look for positive contigious sub-array, here current_sum keeps track of it.
+#In each loop we compare the current_sum with best_sum (i.e Maximum Sum) and update if current becomes larger than best_sum.
+#Time complexity: O(n) and Space Complexity: O(1)
```

Comment:
shift this complexity to the bottom of the file

Reply:
okay, updating it